documentation/infrastructure.md

```diff
@@ -0,0 +1,67 @@
+# Infrastructure diagrams
+
+## CI/CD deployment pipeline
+
+On Pull Request to `master` branch, GitHub Actions `deploy.yml` workflow launches:
+
+```mermaid
```

Comment:
Please add the rendered mermaid diagrams using https://mermaid.ink/

Reply:
Added